crates/proof-of-sql/src/sql/proof/query_proof.rs

```diff
@@ -47,15 +47,15 @@ impl<CP: CommitmentEvaluationProof> QueryProof<CP> {
         accessor: &impl DataAccessor<CP::Scalar>,
         setup: &CP::ProverPublicSetup<'_>,
     ) -> (Self, ProvableQueryResult) {
-        let table_length = expr.get_length(accessor);
-        let num_sumcheck_variables = cmp::max(log2_up(table_length), 1);
-        let generator_offset = expr.get_offset(accessor);
```

Comment:
Shouldn't `get_length` and `get_offset` be removed from the trait since it is now dead code?

Reply:
Yup. I originally wanted to do it in a future PR but can of course do it now.